website/src/components/Hero.tsx

```diff
@@ -55,6 +59,28 @@ function BackgroundIllustration(props) {
   );
 }
 
+function CTA() {
+  const { data: session } = useSession();
+  const router = useRouter();
+  if (session) {
+    router.push("/dashboard");
```

Comment:
Having a redirect buried in a small component feels very weird.  Can you instead do this redirect directly inside of `website/src/pages/index.tsx`?  I think all we need is the session check and redirect.  We don't need a new log in button as we have one in the header already.